packages/cosmic-swingset/Makefile

```diff
@@ -49,7 +49,7 @@ $(EVAL_CLEAN): $(EVAL_CODE)
 	./scripts/clean-core-eval.js $< >$@.t || { rm -f $@.t; exit 1; }
 	mv $@.t $@
 
-RUN_PROTO=../run-protocol
+RUN_PROTO=../inter-protocol
```

Comment:
Why leave RUN_PROTO behind?

Reply:
not intentional. this was a find-replace and I haven't looked over all the instances yet. I figured I would keep putting commits into #5585 but I'll get more into this one and mark it as Draft until I think it's ready to merge.